### PR TITLE
t: Workaround for faulty JSON::Validator error handling

### PR DESCRIPTION
**Pull request by perlpunk**
Issue: https://progress.opensuse.org/issues/75346

JSON::Validator issue: https://github.com/mojolicious/json-validator/issues/231

Includes the regular dependency update.

### CODE REVIEW

**Comment by okurz** [comment]
You should be able to fix the CI problems by either deleting the versions in the ci-packages.txt or rebase on the dependency update PR

@andrii-suse here you can currently see the problem about CI failing when a new Tumbleweed snapshot was released

**Review by Martchus — APPROVED** [review]

File: t/16-utils-job-templates.t [hunk 1]
```diff
@@ -50,6 +51,13 @@ is scalar @$errors, 1, "non existing schema file"
   };
 like($errors->[0], qr{Unable to load schema}, "non existing schema file error message");
 
+$errors = validate_data(schema_file => '/does-not-exist', data => $template);
+is scalar @$errors, 1, "non existing asolute schema file"
```
Comment (kalikiana) [inline t/16-utils-job-templates.t:55]:
```suggestion
is scalar @$errors, 1, "non-existent absolute schema file"
```

File: t/16-utils-job-templates.t [hunk 2]
```diff
@@ -50,6 +51,13 @@ is scalar @$errors, 1, "non existing schema file"
...
+  or do {
+    diag "Error: $_" for @$errors;
+  };
+like($errors->[0], qr{Unable to load schema}, "non existing schema file error message");
```
Comment (kalikiana) [inline t/16-utils-job-templates.t:59]:
```suggestion
like($errors->[0], qr{Unable to load schema}, "non-existent schema file error message");
```

**Review by kalikiana — APPROVED** [review]

File: t/16-utils-job-templates.t (hunk 1 above)
Comment (perlpunk) [reply t/16-utils-job-templates.t]: whoops! fixed

File: t/16-utils-job-templates.t (hunk 2 above)
Comment (perlpunk) [reply t/16-utils-job-templates.t]: ok

**Comment by andrii-suse** [comment]
> @andrii-suse here you can currently see the problem about CI failing when a new Tumbleweed snapshot was released

@okurz again this is because tina never used this ci-packages.txt (since in was created on 14-Okt) to build corresponding cache in the fork. In case if we want to have explicit control over CI environment -  in such cases there is no other solution than running manually `.circleci/build_dependencies.sh` to update ci-packages.txt or take the same in commit from the PR of openqabot.

**Pull request merged**